metaflow/plugins/argo/test/live_run_failure_flow.py

```diff
@@ -0,0 +1,46 @@
+from metaflow import FlowSpec, step
+import time
+
+
+class LiveRunFailureFlow(FlowSpec):
+    """
+    This flow fails. Use this flow to test trigger_live_run function's handling
+    of failure.
+    """
+
+    @step
+    def start(self):
+        """
+        This step waits 5 seconds to let the trigger function test status.
+        """
+        print("LiveRunFailureFlow is starting")
+        time.sleep(5)
```

Comment:
Sleep can be removed since you only test `is_running` against simple flow. The same applies for the other 3 flows.

Reply:
I will remove the 5s wait from flows other than simple and failure flow. Failure flow does test is_running while it's running in the ["test_failure_flow" test](https://github.com/zillow/metaflow/blob/69c48fbe1459521219005d55931ccf57ad5021da/metaflow/plugins/argo/test/test_trigger_live_run.py#L180).